docs/_static/css/pagination.css

```
@@ -0,0 +1,33 @@
+#pagination {
+  display: flex;
+  justify-content: center;
+}
+
+.pagination-previous {
+  margin-right: 2rem;
```

Our linting is going to complain about this because we now expect all styles to be written with logical properties and values. Not really relevant for our docs, but it doesn’t hurt for the docs styles to be written with the same standards.

```suggestion
  margin-inline-end: 2rem;
```